### DISCUSSION TITLE
Is it possible to implement tailwind purge for the css of this site?

### DISCUSSION THREAD

**Discussion by sosukeinu** [jpanther/congo#555] [Q&A] [answered]
How would I go about doing this?
Thank you

**Comment by sosukeinu** [comment, marked as answer]
Nevermind. I found it! Thank you so much for such a great product https://jpanther.github.io/congo/docs/advanced-customisation/#tailwind-configuration

**Reply by jpanther** [reply]
The theme already purges all the unnecessary Tailwind classes so this is not strictly necessary, but if you do build yourself you will achieve essentially the same result.
